modules/gitops-release-notes-1-6-0.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-6-0_{context}"]
+= Release notes for {gitops-title} 1.6.0
+
+{gitops-title} 1.6.0 is now available on {product-title} 4.8, 4.9, and 4.10.
+
+[id="new-features-1-6-0_{context}"]
+== New features
```

Comment:
For IBM Z and IBM Power support please add to the new features: 

This release of {gitops-title} adds support for IBM Z and IBM Power on {product-title} 4.10.